docs/whirlpool/docs/02-Architecture Overview/07-TokenExtensions Support.md

```diff
@@ -0,0 +1,161 @@
+# TokenExtensions
+
+At slot `268396603` (May 28, 2024 07:08:16 UTC), Whirlpool program have been upgraded to support TokenExtensions (aka Token-2022 program).
+
+The same program has been deployed on Devnet. So you can try it on Devnet.
```

Comment:
Think we can just remove this line given that we no also have eclipse. Maybe we can say the same program version is deployed on all chains that Orca supports or something but not really necessary